src/test/java/com/intuit/graphql/orchestrator/fieldresolver/FieldResolverBatchSelectionSetSupplierLiteralsTest.java

```diff
@@ -94,11 +95,20 @@ public void testWithObjectLiteralsArgument() {
             new ResolverArgumentDefinition("petIdInputObject", "{ id : \"$petId\" }", objectType)
         ));
 
+        FieldResolverContext fieldResolverContext = FieldResolverContext.builder()
+                .parentTypeDefinition(testFieldResolverContext.getParentTypeDefinition())
+                .fieldDefinition(fieldDefinitionWithResolver)
+                .requiresTypeNameInjection(true)
+                .serviceNamespace("TESTSVC")
+                .resolverDirectiveDefinition(resolverDirectiveDefinitionMock)
+                .requiredFields(testDFEDataSource.keySet())
+                .build();
+
         String[] resolverSelectedFields = new String[] {"petById"};
 
         // WHEN
         subject = new FieldResolverBatchSelectionSetSupplier(resolverSelectedFields, dataFetchingEnvironments,
-            testFieldResolverContext);
+            fieldResolverContext);
```

Comment:
I think it is ok to re-use the variable `testFieldResolverContext` since it is class level

Reply:
The context needs to have the requiredFields portion built. If I set that variable it will be done for all the other tests, which would give a bad representation because they would not have that field or would have a different set in their case.

Reply:
I mean the new FieldResolverContext object created in this test method, can be assigned to `testFieldResolverContext`.  The other tests will not be affected.

Reply:
Ah yep you are right....I forgot that method with before annotation should set that object again